.azure/pipelines/devBuilds.yml

```diff
@@ -10,7 +10,7 @@ pr: none
 
 # Schedule this pipeline to run every midnight
 schedules:
-- cron: "0 0 * * *"
+- cron: "0 8 * * *"
```

Comment:
Will this change to 7 in the summer 😺

Reply:
No :) An hour of difference is fine. I just want to make sure it runs sometime near midnight to make sure we're not `stealing` agents from PR builds.